versions/latest/modules/en/pages/cluster-admin/manage-clusters/access-clusters/add-users-to-clusters.adoc

```diff
@@ -1,14 +1,12 @@
 = Adding Users to Clusters
 
-If you want to provide a user with access and permissions to _all_ projects, nodes, and resources within a cluster, assign the user a cluster membership.
+To provide a user access to view certain cluster-level resources and create new projects, assign the user a **Cluster Membership**. Cluster members can create projects and manage resources in those projects. However, not all resources, namespaces and workloads in a cluster are accessible by cluster members.
 
 [TIP]
 ====
-
-Want to provide a user with access to a _specific_ project within a cluster? See xref:cluster-admin/project-admin/add-users-to-projects.adoc[][Adding Project Members] instead.
+Want to provide a user with access to a _specific_ project within a cluster? See xref:../../../cluster-admin/project-admin/add-users-to-projects.adoc[Adding Project Members] instead.
```

Comment:
Per previous discussions, we should use [Antora's linking style](https://docs.antora.org/antora/latest/page/page-links/) and not reintroduce relative links with `../*`.